include/irr/asset/IVirtualTexture.h

```diff
@@ -751,6 +762,22 @@ class IVirtualTexture : public core::IReferenceCounted, public IVirtualTextureBa
 
     virtual SViewAliasTextureData createAlias(const SMasterTextureData& _addr, E_FORMAT _viewingFormat, const IImage::SSubresourceRange& _subresRelativeToMaster) = 0;
 
+    //! @returns pointer to reserved space for allocators
+    uint8_t* copyVirtualSpaceAllocatorsState(uint32_t _count, pg_tab_addr_alctr_t* _dstArray)
+    {
+        _count = std::min(_count, m_pageTable->getCreationParameters().arrayLayers);
+        const uint32_t bufSz = m_pageTableLayerAllocators[0].get_total_size();
+        const uint32_t resSpcPerAlctr = pg_tab_addr_alctr_t::reserved_size(m_pageTableLayerAllocators[0].get_total_size(), m_pageTableLayerAllocators[0]);
```

Comment:
doesn't `reserved_size` take the argumetns the other way around?

Reply:
nah, at least not in this allocator